### ISSUE TITLE
Enhance accessibility on the Short Publication Page

### ISSUE THREAD

**Issue by edwinmp** [devinit/DIwebsite-redesign#1285]
Perform an accessibility audit on the Short Publications Page and fix any flagged accessibility issues.

Resources: [WAVE](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

For full coverage, you'll need to create a test page that makes use of every possible StreamField item.

**Comment by edwinmp** [comment]
Hi @EmmawDI ... We've made accessibility enhancements for the Short Publications page and this has been deployed to [staging](https://dev.devinit.org/). Please review when u have a moment.

**Comment by edwinmp** [comment]
> Do you have a record of what changes were made?

@davidebukali ^^


> It seems as though text contrast is a recurring problem, especially with linked text. Do you think this is something we need to look to update across the site?

Yes, some of our colours have been flagged as not being of great contrast, not just on text. To ensure uniformity, it's not a change we should make in bits if it's happening all over the site.

**Comment by EmmawDI** [comment]
Sure, that makes sense. Simon is going to talk to Anna about updating some colours across the site so will get back to you asap on that

**Comment by davidebukali** [comment]
Yes we have a record of the changes which were mostly to the css. A few updates were made to the structure of html. These could apply to other publication types since they share sections.

**Comment by EmmawDI** [comment]
Anna is happy for us to make colour updates necessary to meet accessibility requirements. 
- Hyperlinked text (including "share" links and notes) and pull-out quote text: if this is changes to Colour 5 of the respective hue, we should be meeting the [WCAG 1.4.6 AAA criteria for enhanced contrast at 7:1. ](https://www.w3.org/TR/WCAG21/#contrast-enhanced)
- More decorative elements (ie white text on coloured boxes for CTAs, certain buttons, table headings etc) will also need to be made darker, changing to colour 5 across the board. Note the default colour of the pages doesn't need to change from Colour 1 as these are brand colours and do not have text directly on them. 

Would it be possible for this change to be made in staging, for us to review and present to Anna? As you mentioned, these changes will affect the whole site so we want to avoid any situation where one page type looks noticeably different from another. 

On a broader note - what is the plan going forward to continue the audit? Simon has flagged the [pattern library](https://development-initiatives.surge.sh/page-templates/) to me, which we could use as a basis to record issues found and fixes implemented, so we can see crossover between page types. 

Thank you for your work on this so far and do let me know if you need clarification on any of the above.

**Comment by k8hughes** [comment]
Hi Emma, can you speak to Anna about what we are doing budget wise for this update?  We don't have a code for over arching website work this year so I need to know where we charge dev time.

**Comment by EmmawDI** [comment]
Hi Kate - raised this with Anna. She said: "ideally we should put together a ToR setting out critical website work needed this year with a budget next to it. I've put into the external relations budget a range of costs for web development although that hasn't been approved yet, but it includes accessibility training for the devs, a % of their time for things like updates and changes but nothing too major. So if we can write up what we need, they estimate the costs, we can then check that against what I've submitted, and if need be, push for more to be added in any future reforecasting as and when money comes available." Does that help at all?

**Comment by k8hughes** [comment]
So it sounds like we can't approve work to be done at the moment but we have a process to follow to get there?

**Comment by EmmawDI** [comment]
Agreed, but I think any estimates we might be able to come up with for how much time/money would be needed for the work would help

**Comment by k8hughes** [comment]
If you can share that TOR of the website work you need then we can estimate against it and schedule the work in.
